jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -237,20 +237,106 @@ def test_wcslink_rotated(self):
         assert self.viewer.label_mouseover.value == '+1.00000e+00 '
         assert self.viewer.label_mouseover.world_ra_deg == ''
         assert self.viewer.label_mouseover.world_dec_deg == ''
+        assert self.viewer.label_mouseover.unreliable_world
+        assert self.viewer.label_mouseover.unreliable_pixel

```

Comment:
This is what I would have expected. This visible data has no WCS, so it follows FITS WCS data that should be the reference and that one does not have a bounding box. I'll investigate.
```suggestion
        assert not self.viewer.label_mouseover.unreliable_world
        assert not self.viewer.label_mouseover.unreliable_pixel
```

Reply:
Found a bug. I will push out fix in a subsequent commit here.